src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -238,7 +238,94 @@ public Task Get()
                 }
             }
         }
-        
+
+        [RavenAction("/admin/config-expiration-bundle", "POST", "/admin/config-expiration-bundle?name={databaseName:string}")]
```

Comment:
This is a raft command it needs the access to server store as much as i understand the per database handler do not have access to it